### PR TITLE
PMM-7 Add PMM 2.29.1

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/pmm.2.29.1.pmm-server.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            "status": "available",
+            "critical": false
+          },
+          "1.10.0": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0",
+            "image_hash": "73d2266258b700a691db6196f4b5c830845d34d57bdef5be5ffbd45e88407309",
+            "status": "recommended",
+            "critical": false
+          }
+        },
```
Comment (nmarukovich) [inline sources/pmm.2.29.1.pmm-server.json:20]: do we need 1.11.0 operator version here?

**Review by gen1us2k — APPROVED** [review]

File: sources/pmm.2.29.1.pmm-server.json (hunk 1 above)
Comment (BupycHuk) [reply sources/pmm.2.29.1.pmm-server.json:20]: @beatahandzelova have we tested PXC 1.11.0 with the latest PMM?

Comment (BupycHuk) [reply sources/pmm.2.29.1.pmm-server.json:20]: @nmarukovich let's merge as is and if it will be required add 1.11.0 for both 2.29.0 and 2.29.1 later

Comment (beatahandzelova) [reply sources/pmm.2.29.1.pmm-server.json:20]: @BupycHuk yes

**Review by tplavcic — APPROVED** [review]

**Pull request merged**